apps/authenticator/lib/sessions/tokens/authorization_code.ex

```diff
@@ -0,0 +1,36 @@
+defmodule Authenticator.Sessions.Tokens.AuthorizationCode do
+  @moduledoc """
+  Access token configurations.
+  """
+
+  use Joken.Config
+
+  add_hook Joken.Hooks.RequiredClaims, ~w(exp iat nbf iss aud azp jti sub typ scope)
+  add_hook Authenticator.Sessions.Tokens.Hooks.ValidateUUID, ~w(sub aud)
+
+  # Two hours in seconds
+  @max_exp 60 * 60 * 2
```

Comment:
```suggestion
  @exp_in_seconds 60 * 60 * 2
```

Time variables should contain the unit on it's name, so we don't need a comment for it